### ISSUE TITLE
Reconsider wetkit multilingualism depending on i18n_sync

### ISSUE THREAD

**Issue by joel-osc** [wet-boew/wet-boew-drupal#792]
We are currently working on a couple of sites based on this distro and have decided to use Entity Translation for nodes. I thought it would be a good idea to turn off i18n_sync since it was no longer needed, but could not because wetkit multilingualism depends on it. Maybe in the future as Entity Translation becomes the norm for multilingual sites we should consider removing this dependency.

**Comment by joejoseph00** [comment]
Have you put any sites into production using Entity ?

**Comment by joel-osc** [comment]
One using it on taxonomy and it will be used from here on in on everything.

**Comment by sylus** [comment]
i18n sync is now turned off by default as rolling with ET throughout.
